### PR TITLE
implement core reducers and handlers

### PR DESCRIPTION
**Pull request by jkirkpatrick24**
Implements methods for creating reducers.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    "url": "https://github.com/autotelic/redux-core-reducers.git/issues"
+  },
+  "homepage": "https://github.com/autotelic/redux-core-reducers.git",
+  "license": "MIT",
+  "dependencies": {
+    "lodash.get": "^4.4.2",
+    "seamless-immutable": "^7.1.4"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
```
Comment (averagebeard) [inline package.json]: Why not upgrading to babel 7?

File: src/handlers.test.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    };
+
+    keys = {
+      name: 'otherName',
+      value: 'otherValue',
+    };
+  });
+
+  describe('nameValueHandler', () => {
+    it('uses the defauly keys to to update the state', () => {
```
Comment (averagebeard) [inline src/handlers.test.js]: [typo]
default

File: package.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@autotelic/redux-core-reducers",
+  "version": "0.1.0",
+  "description": "A collection of re-usable and configurable reducers redux applications",
```
Comment (eadmundo) [inline package.json]: [nit] "...reducers redux..." -> "...reducers for redux..."?

**Pull request merged**